GitCommands/Git/GitCommandHelpers.cs

```diff
@@ -1018,6 +1018,66 @@ public static IReadOnlyList<GitItemStatus> GetSkipWorktreeFilesFromString(string
             return result;
         }
 
+        /// <summary>
+        /// Parse the output from --tool-help
+        /// </summary>
+        /// <param name="output">The output string</param>
+        /// <returns>list with tool names</returns>
+        public static List<string> ParseCustomDiffMergeTool(string output)
+        {
+            var tools = new List<string>();
+
+            // Simple parsing of the textual output opposite to porcelain format
+            // Explanation of the modes:
+            //  0: Initial mode (to detect first line, direct transit to next mode)
+            //  1: Built in tools, in the path (first is header, ended by a newline)
+            //  2: Newlines separating tools
+            //  3: Built in tools, in the path (first is header, ended by a newline)
+            //  4: Newlines separating tools
+            //  5: Unavailable tools, skipped
+            //  6: End note, skipped
+            //  5: Newlines
```

Comment:
> > There's no need to hide easter eggs for me, e.g. in the nevertheless very helpful comment. 😉

> Do you always want a mention for issues you are already reviewing?

No, I meant just the minor inconsisties in the comment above. I didn't want to request a change.

Afterwards, I had a look at the screenshots again.